notebooks/book2/11/mcmc_traceplots_unigauss.ipynb

```diff
@@ -0,0 +1,965 @@
+{
```

Comment:
<p>We initialised <code>prior_alpha</code> in the initial cells and now we're using <code>log_prob_alpha</code> etc. Should we reuse variables?   </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/908/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Yes, done.</p>